web/cobrands/sass/_base.scss

```diff
@@ -1326,12 +1326,23 @@ input.final-submit {
   }
 }
 
-.item-list--wards__item {
+.item-list--wards {
   @extend %item-list__item--link;
-
+  padding: 1em;
   label {
-    margin: 0;
-    padding: 0.5em 1em;
+    margin-top: inherit;
```

Comment:
Don't think this is doing anything? It's overridden by the margin-top: -2px of the govuk style, unless I've missed something.